client/components/jetpack/daily-backup-status/status-card/backup-just-completed.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ */
+import { useTranslate } from 'i18n-calypso';
+import { Moment } from 'moment';
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { preventWidows } from 'calypso/lib/formatting';
+import { INDEX_FORMAT } from 'calypso/lib/jetpack/backup-utils';
+import { backupMainPath } from 'calypso/my-sites/backup/paths';
+import getSelectedSiteSlug from 'calypso/state/ui/selectors/get-selected-site-slug';
+import useGetDisplayDate from '../use-get-display-date';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+import cloudScheduleIcon from './icons/cloud-schedule.svg';
+
+const BackupJustCompleted: React.FC< Props > = ( { justCompletedBackupDate, lastBackupDate } ) => {
+	const translate = useTranslate();
+	const siteSlug = useSelector( getSelectedSiteSlug );
+	const getDisplayDate = useGetDisplayDate();
+
+	const justCompletedDisplayDate = getDisplayDate( justCompletedBackupDate, false );
+	const lastBackupDisplayDate = getDisplayDate( lastBackupDate, false );
```

Comment:
In `client/components/jetpack/backup-card/backup-just-completed.tsx`, we apply to `lastBackupDate` the `useDateWithOffset` hook before using the variable. Here we don't do that. Could we be inadvertently showing two different dates depending on the version of the component?

Reply:
This is a good catch! I've written the components in both variants to assume the passed-in date is a Moment, which means when they get processed by `getDisplayDate`, they're converted to the site's timezone automatically. So, in short, my extra `useDateWithOffset` call is redundant and not needed. I'll remove it immediately. :-)

Reply:
!! On second thought, I can't remove it completely. This is because further down in the component we build a URL with a `date` parameter, which needs to account for the site's timezone.